### PR TITLE
feat(msp): add reset-button

### PR DESCRIPTION
**Pull request by wuhuiluo**
## What this PR does / why we need it:
add reset-button

## Does this PR introduce a user interface change?

截屏2021-08-09上午11 36 51
✅ Yes(screenshot is required)

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |      [monitoring] link debugging, add reset button and clear all contents of the page        |
| 🇨🇳 中文    | 【监控】链路调试，添加重置按钮，清空页面所有内容         |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Comment by daskyrk** [comment]
add changeLog

File: shell/app/modules/msp/monitor/trace-insight/pages/trace-querier/trace-querier.tsx [hunk 1]
```diff
@@ -197,10 +202,21 @@ const TraceInsightQuerier = () => {
...
           </Col>
-          <Col span={3}>
+          <Col span={6}>
             <Button type="primary" loading={isRequestTraceFetching} onClick={handleRequestTrace}>
               {i18n.t('msp:request')}
             </Button>
+            <Button className="ml-4" danger onClick={() => setModalVis(true)}>
+              {i18n.t('common:reset')}
+            </Button>
+            <Modal
```
Comment (Zero-Rock) [inline shell/app/modules/msp/monitor/trace-insight/pages/trace-querier/trace-querier.tsx:212]: Modal.confirm is better

Comment (daskyrk) [reply shell/app/modules/msp/monitor/trace-insight/pages/trace-querier/trace-querier.tsx:212]: image
Use Popconfirm in such condition.

File: shell/app/modules/msp/monitor/trace-insight/pages/trace-querier/trace-querier.tsx [hunk 2]
```diff
@@ -197,10 +202,21 @@ const TraceInsightQuerier = () => {
               />
             </FormItem>
           </Col>
-          <Col span={3}>
+          <Col span={6}>
             <Button type="primary" loading={isRequestTraceFetching} onClick={handleRequestTrace}>
               {i18n.t('msp:request')}
             </Button>
+            <Button className="ml-4" danger onClick={() => setModalVis(true)}>
```
Comment (daskyrk) [inline shell/app/modules/msp/monitor/trace-insight/pages/trace-querier/trace-querier.tsx:209]: This is not deletion operation, use normal button is ok.

**Pull request merged**